.github/scripts/split_spec.sh

```diff
@@ -17,4 +23,4 @@ echo "Moving preview files to preview and private-preview folder"
 find ./openapi/v2 -type f -name "*private-preview*" -exec mv -f {} ./openapi/v2/private/ \;
 
 echo "Generate the versions.json file for private preview APIs"
-foascli versions --spec ./openapi/v2.json --stability-level PRIVATE-PREVIEW -o ./openapi/v2/private/versions.json --env "${target_env:?}"
+foascli versions --spec ./openapi/.raw/v2.json --stability-level PRIVATE-PREVIEW -o ./openapi/v2/private/versions.json --env "${target_env:?}"
```

Comment:
Could you filter out from the branded preview the v2 spec and the v2 in the .raw?

Reply:
not sure I follow, but for the branded preview, I made it use the filtered version, since it would hide the hidden contents

Reply:
There is a step in the release process where we create the branded preview (branded_preview.sh), and I see that right now https://htmlpreview.github.io/?https://github.com/mongodb/openapi/blob/main/openapi/branded-preview.html we list v2.json and v2.yaml. I am wondering if with these changes we will start to see also the spec in .raw 🤔  This can also be a follow up ticket. Thanks!

Reply:
ohhh got it! good point, not sure and I hope not :D If we see it I'll remove in a follow-up